From 753701ca1fc2b9a0cca97879eed3f5345e0a278f Mon Sep 17 00:00:00 2001
From: bbhtt <bbhtt.zn0i8@slmail.me>
Date: Sun, 14 Jan 2024 13:44:40 +0530
Subject: [PATCH] Don't compare module paths with QtCore base path

In flatpak, Qtcore is in /usr but individual modules may not be part of
runtime (/usr) which is read only.

Individual Qt modules can be in `/app`
---
 sources/pyside6/cmake/PySideHelpers.cmake | 2 +-
 1 file changed, 1 insertion(+), 1 deletion(-)

diff --git a/sources/pyside6/cmake/PySideHelpers.cmake b/sources/pyside6/cmake/PySideHelpers.cmake
index c728e7c43..e6911e958 100644
--- a/sources/pyside6/cmake/PySideHelpers.cmake
+++ b/sources/pyside6/cmake/PySideHelpers.cmake
@@ -230,7 +230,7 @@ macro(collect_module_if_found shortname)
     # If the module was found, and also the module path is the same as the
     # Qt5Core base path, we will generate the list with the modules to be installed
     set(looked_in_message ". Looked in: ${${_name_dir}}")
-    if("${${_name_found}}" AND (("${found_basepath}" GREATER "0") OR ("${found_basepath}" EQUAL "0")))
+    if("${${_name_found}}")
         message(STATUS "${module_state} module ${name} found (${ARGN})${looked_in_message}")
         # record the shortnames for the tests
         list(APPEND all_module_shortnames ${shortname})
-- 
2.43.0

